solvebio/resource/object.py

```diff
@@ -0,0 +1,69 @@
+"""Solvebio Object API resource"""
+from ..help import open_help
+from ..client import client
+
+from .solveobject import convert_to_solve_object
+from .apiresource import CreateableAPIResource
+from .apiresource import ListableAPIResource
+from .apiresource import SearchableAPIResource
+from .apiresource import UpdateableAPIResource
+from .apiresource import DeletableAPIResource
+
+
+class Object(CreateableAPIResource,
```

Comment:
Yes that seems fine to me. We should create tickets with features and enhancements that we think mirror what you can do in the UI (and more). Lets evaluate the client today and do some QA, test it out a bit.